docs/integration/alerts-telegram-integration.md

```diff
@@ -0,0 +1,85 @@
+title: Telegram Alerts Integration
+description: Sematext threshold, anomaly and / or heartbeat Alerts integration with Telegram messenger. Use them together to get the information about each notification quickly and on any device.
+
+## In Telegram
+
+**1.** Create a Telegram account at [telegram.org](https://telegram.org).
+
+**2.** Setup your Telegram client and start chatting with a user named **BotFather**. This is a place where you can create your bot. Just type in:
+
+```
+/newbot
+```
+
+And in the next line type your bot name, i.e. **SematextNotificationBot**. 
+
+Finally, type the bot username, it has to end with the **_bot** suffix, i.e. **SematextNotificationBot_bot**.
+
+If everything went successfully you should get a message with the **Bot Token API key**:
+
+<img class="content-modal-image" alt="Create Telegram Bot Token API Key" src="../../images/integrations/create-telegram-integration_bot_key.png" title="Create Telegram Bot Token API Key">
+
+Note the **Bot Token API key** down, you will need to provide it in Sematext UI.
```

Comment:
Note the **Bot Token API key** down, you will need to provide it in Sematext UI.
==>
Copy the **Bot Token API key** somewhere, you will need it to configure the notification hook in Sematext.

Reply:
Done.